### ISSUE TITLE
Nostr relay

### ISSUE THREAD

**Issue by jjmmbb** [sahilph/umbrel-HTTPSizer#5]
Clients can not connect to Nostr Relay anymore.

**Comment by sahilph** [comment]
I have not used that app. But based Umbrel's on the documentation it seems that you have to use "ws://umbrel:4848" for the connection.

So in this case, you will have to use "**wss**://umbrel:4848" for the secure connection, have you tired that ?

**Comment by jjmmbb** [comment]
> I have not used that app. But based Umbrel's on the documentation it seems that you have to use "ws://umbrel:4848" for the connection.
> 
> 
> 
> So in this case, you will have to use "**wss**://umbrel:4848" for the secure connection, have you tired that ?

Yes. I have tried both. ws:// doesn't work anymore and wss:// don't connect. I think that is a problem caused by warning about the certificate.

**Comment by sahilph** [comment]
Are you using tailscale to access it, if yes then this most likely is related to #3 and should work when that is completed.

Temporarily, you can just set the clients to ignore the SSL errors. 
On the client devices that you are using, is there any way for them to ignore SSL errors ?

**Comment by jjmmbb** [comment]
> Temporarily, you can just set the clients to ignore the SSL errors. On the client devices that you are using, is there any way for them to ignore SSL errors ?

I have tried on local addresses different from tail and unfortunatelly Nostr Clients doesn't have an option to ignore SSL errors.

**Comment by sahilph** [comment]
Has the new update solved this Nostr relay issue ?
